### ISSUE TITLE
Train CED models with two step training

### ISSUE THREAD

**Issue by radka-j** [alan-turing-institute/ARC-MTQE#56]
Use single random seed, pre-train for 100 epochs  with the following data:
- [x] DEMETR 
    - [x] base model pre-training
    - [x]  fine-tuning
- [x] Multilingual authentic
    - [x] base model pre-training
    - [x]  fine-tuning
- [x] DEMETR + multilingual authentic
    - [x] base model pre-training
    - [x]  fine-tuning
- [x] En-De synthetic from WMT22
    - [x] base model pre-training
    - [x]  fine-tuning
- [x] En-De synthetic from WMT22 (small version ?)
    - [x] base model pre-training
    - [x]  fine-tuning
    
 Save best checkpoint based on average  performance on all validation data.
